mesheryctl/internal/cli/root/pattern/offboard.go

```diff
@@ -53,24 +53,24 @@ mesheryctl app offboard -f [filepath]
 			return nil
 		}
 
-		app := ""
+		pattern := ""
 		isID := false
 		if len(args) > 0 {
-			app, isID, err = utils.ValidId(mctlCfg.GetBaseMesheryURL(), args[0], "pattern")
```

Comment:
Move into cobra.ARgs and return Meshkit error.

Reply:
Solved.